packages/matchbox-icons/icons/index.js

```diff
@@ -0,0 +1,960 @@
+/* eslint-disable max-lines */
```

Comment:
So this and other utilities don't get buried, what do you think about making about this structure?

```
src/index.js
src/IconBase/
src/icons/*.js
```